src/app/app.component.ts

```diff
@@ -88,38 +91,42 @@ export class AppComponent implements OnInit, OnDestroy {
   }
 
   loadCmsSeries(auth: HalDoc) {
-    auth.followItems('prx:series', {filters: 'v4', zoom: 'prx:distributions'}).subscribe((series: HalDoc[]) => {
-      if (series.length === 0) {
-        this.error = 'Looks like you don\'t have any podcasts.';
+    auth.followItems('prx:series',
+      {per: auth.count('prx:series'), filters: 'v4', zoom: 'prx:distributions'}).subscribe((series: HalDoc[]) => {
+      if (series && series.length === 0) {
+        this.store.dispatch(new CmsPodcastsAction({podcasts: []}));
```

Comment:
the initial state of the podcasts store is now undefined so that "you don't have any podcasts" error handling watches for empty array